site/docs/latest/functions/api.md

```diff
@@ -2,9 +2,70 @@
 title: The Pulsar Functions API
 ---
 
+Pulsar Functions provides an easy-to-use API that develoeprs can use to 
+
+## Core programming model
+
+Pulsar Functions
+
+### Source and sink topics
+
+All Pulsar Functions have one or more **source topics** that supply messages to the function.
+
+### Sink topic
+
+At the moment, Pulsar Functions can have at most one **sink topic** to which processing results are published.
+
```

Comment:
You can add that inside the function logic, it can specifically publish to as many topics as it wants.